src/game/mario_actions_cutscene.c

```diff
@@ -582,15 +582,27 @@ s32 act_debug_free_move(struct MarioState *m) {
 }
 
 void general_star_dance_handler(struct MarioState *m, s32 isInWater) {
+    struct Object *celebStar = NULL;
+
     if (m->actionState == ACT_STATE_STAR_DANCE_CUTSCENE) {
         switch (++m->actionTimer) {
             case 1:
-                spawn_object(m->marioObj, MODEL_STAR, bhvCelebrationStar);
+                celebStar = spawn_object(m->marioObj, MODEL_STAR, bhvCelebrationStar);
+#ifdef CELEBRATION_STAR_COPY_MODEL
```

Comment:
Just making a note here so changing this ifdef to match the define name doesn't get forgotten.